src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(int length, int width, String color) {
+        super(color);
+        this.length = length;
+        this.width = width;
+    }
+
+    @Override
+    public double getArea() {
+        return length * width;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: rectangle, area: "
+                + getArea()
+                + " sq. units, length: "
+                + length
+                + " units, width: "
+                + width
+                + " units, color: "
+                + getColor());
```

Comment:
Avoid using System.out.println for drawing the figure. Consider abstracting the drawing logic to a separate method or class that handles the presentation logic.